src/senaite/core/content/sampletype.py

```diff
@@ -0,0 +1,435 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of SENAITE.CORE.
+#
+# SENAITE.CORE is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright 2018-2024 by it's authors.
+# Some rights reserved, see README and LICENSE.
+
+from AccessControl import ClassSecurityInfo
+from bika.lims import api
+from bika.lims import senaiteMessageFactory as _
+from bika.lims.interfaces import IDeactivable
+from datetime import timedelta
+from plone.autoform import directives
+from plone.supermodel import model
+from Products.CMFCore import permissions
+from senaite.core.api.dtime import dict_to_timedelta
+from senaite.core.api.dtime import timedelta_to_dict
```

Comment:
Quite often, we use the module name when using functions from `api`-like modules instead of explicitly import the function, e.g.:

```python
from senaite.core.api import dtime

...

value = dtime.timedelta_to_dict(delta)
```